shelley/chain-and-ledger/executable-spec/test/Generator/LedgerTrace/QuickCheck.hs

```diff
@@ -32,7 +32,8 @@ instance TQC.HasTrace LEDGER Word64 where
               <*> genPParams
               <*> genCoin 0 1000
 
+  -- TODO pass core node keys as inputs
   sigGen _ ledgerEnv ledgerSt =
-    genTx ledgerEnv ledgerSt traceKeyPairs traceVRFKeyPairs
+    genTx ledgerEnv ledgerSt traceKeyPairs [] traceVRFKeyPairs
```

Comment:
Does this cause the QC generator to only generate empty lists of `[CoreKeyPair]`s?

Reply:
yup! good catch thanks ;)